<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pdfGuiClass</class>
 <widget class="QMainWindow" name="pdfGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>659</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>33</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pdfGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="fileFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="openFile">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/images/open.ico);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/open.ico</normaloff>images/open.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="toolTip">
          <string>Save as</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/save.ico</normaloff>images/save.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsButton">
         <property name="toolTip">
          <string>save copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/savecopy.ico</normaloff>images/savecopy.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveEncodedButton">
         <property name="toolTip">
          <string>save decoded</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/decrypt_icon.jpg</normaloff>images/decrypt_icon.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="contentButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/content.ico</normaloff>images/content.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="textButton">
         <property name="toolTip">
          <string>Show panel witj text working</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/setText.ico</normaloff>images/setText.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="imageWork">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/images.ico</normaloff>images/images.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="opSelect">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/operator.jpg</normaloff>images/operator.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="fileProp" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="contentFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="derotate">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/derotate.ico</normaloff>images/derotate.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotate">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/rotate.ico</normaloff>images/rotate.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="insertEmpty">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/addpage.png</normaloff>images/addpage.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pageUp">
            <property name="text">
             <string>up</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/uppage.ico</normaloff>images/uppage.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pageDown">
            <property name="toolTip">
             <string>page down</string>
            </property>
            <property name="text">
             <string>down</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/downpage.ico</normaloff>images/downpage.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deletePage">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/deletepage.ico</normaloff>images/deletepage.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getText">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/TextConvert.ico</normaloff>images/TextConvert.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="textFrame">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>Panel with text tools</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>300</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="selectTextButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/hand.ico</normaloff>images/hand.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertTextButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/insert_text.ico</normaloff>images/insert_text.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/deleteText.ico</normaloff>images/deleteText.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="erase">
         <property name="text">
          <string>erase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="highlightButton">
         <property name="text">
          <string>highlight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searcher">
         <property name="text">
          <string>search/replace</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="OpenPdf" name="openedPdfs">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenPdf</class>
   <extends>QTabWidget</extends>
   <header location="global">openpdf.h</header>
   <container>1</container>
   <slots>
    <slot>openAnotherPdf()</slot>
    <slot>closeAndRemoveTab(int)</slot>
    <slot>createNewRevision()</slot>
    <slot>save()</slot>
    <slot>saveAs()</slot>
    <slot>insertEmpty()</slot>
    <slot>insertRange()</slot>
    <slot>deletePage()</slot>
    <slot>display()</slot>
    <slot>rotate(int)</slot>
    <slot>print()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pdfgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>getText</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>getText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openedPdfs</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>openedPdfs</receiver>
   <slot>closeAndRemoveTab(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openFile</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>openAnotherPdf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
<connection>
   <sender>insertTextButton</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>setModeInsertText()</slot>
</connection>
<connection>
   <sender>selectTextButton</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>setModeSelectText()</slot>
</connection>
  <connection>
   <sender>saveAsButton</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveEncodedButton</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>saveEncoded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insertEmpty</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>insertEmpty()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deletePage</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>deletePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contentButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>contentFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate</sender>
   <signal>clicked()</signal>
   <receiver>openedPdfs</receiver>
   <slot>rotate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contentButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileProp</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
